catalog/ui/src/app/Catalog/CatalogItemForm.tsx

```diff
@@ -326,7 +326,7 @@ const CatalogItemFormData: React.FC<{ namespace: string; catalogItemName: string
                     <p>Setup a user interface for the workshop attendants to access their credentials.</p>
                   ) : (
                     <ul>
-                      <li>- Provision independent services for each seat in the workshop.</li>
+                      <li>- Provision independent services for each attendant in the workshop.</li>
```

Comment:
"attendant" is the wrong word here. An "attendant" would read like a "flight attendant", it reads as someone who "attends to" the workshop in the same that a flight attendant takes care of people's experience on a flight.

The word I think we should have is "attendee", such as: "Provision independent services for each workshop attendee." This would imply someone who "attends" the workshop, that is someone who merely shows up to it. The English language is crazy.